lib/NormalModule.js

```diff
@@ -673,9 +678,11 @@ class NormalModule extends Module {
 		return rule.test(content);
 	}
 
-	// check if module should not be parsed
-	// returns "true" if the module should !not! be parsed
-	// returns "false" if the module !must! be parsed
+	/**
+	 * @param {string | Function | RegExp | [RegExp, ...RegExp[]] | [string, ...string[]]} noParseRule the parse-rule
```

Comment:
please reference the type generated from schema here

Reply:
Could you please elaborate?

Reply:
please reference the type generated from schema here